src/libs/OptionsListUtils.ts

```diff
@@ -2607,8 +2610,13 @@ function getEmptyOptions(): Options {
     };
 }
 
-function shouldUseBoldText(report: ReportUtils.OptionData): boolean {
-    return report.isUnread === true && ReportUtils.getReportNotificationPreference(report) !== CONST.REPORT.NOTIFICATION_PREFERENCE.MUTE;
+function shouldUseBoldText(report: ReportUtils.OptionData, shouldBoldHiddenChat = true): boolean {
+    const notificationPreference = ReportUtils.getReportNotificationPreference(report);
```

Comment:
@tgolen Can you please confirm here? If we open a hidden chat. It will be displayed in LHN as the focused report. Then if we mark it as unread, should we display this as bold in LHN?